### PR TITLE
Fixed YAML markup for geometry2 bugs

### PR DESCRIPTION
**Pull request by ChrisTimperley**

### CODE REVIEW

**Comment by ipa-hsd** [comment]
Tested it for `3ec0485.bug`
```
Error validating data '../geometry2/3ec0485/3ec0485.bug' with './robust.yaml'
	
	failure-codes: Required field missing
	fault-codes: Required field missing
	bug.package: 'ros/geometry2/tf2_ros' is not a list.
	bug.reported-by: 'member-developer' not in ('guest user', 'contributor', 'member developer', 'automatic', 'unreported')
	bug.time-reported: 'N/A' is not a date.
	bug.time-reported: 'N/A' is not a null.
	fix.license: 'BSD' is not a list.
	fix.time: '2015-05-10 21:11 GMT+2' is not a date.
	fix.time: '2015-05-10 21:11 GMT+2' is not a null.
```

**Comment by ChrisTimperley** [comment]
Fixed licenses with:
```
find . -name *.bug | xargs -n1 sed -i "s/license: BSD/license: ['BSD']/g"
```
I manually checked with `grep` that the command above wouldn't break the files in other places:
```
grep "license: BSD" . -r
```

File: geometry2/f2568f5/f2568f5.bug [hunk 1]
```diff
@@ -45,7 +46,7 @@ fix:
   fix-in:
     - tf2_sensor_msgs/package.xml
   languages:
-    - package.xml
+    - Package XML
```
Comment (ipa-hsd) [inline geometry2/f2568f5/f2568f5.bug:49]: Have these new values been noted somewhere?

Comment (ChrisTimperley) [reply geometry2/f2568f5/f2568f5.bug:49]: They haven't. I'm updating these values across the dataset for the sake of the analysis at the moment. I'm only including changes to `geometry2` in this PR. Below is a count of the frequency of each value for `fix.languages`:
```
Counter({'C++': 110, 'CMake': 26, 'Python': 23, 'Package XML': 23, 'Xacro/URDF': 18, 'XML': 8, 'Launch XML': 7, 'YAML': 5, 'C': 4, '+': 4, 'python': 4, 'launch XML': 4, 'URScript': 3, 'srv': 1, 'collada': 1, 'shell': 1, 'ASCII STL': 1, 'yaml': 1, 'RViz': 1, 'ROS Message': 1, 'udev rules': 1, 'ROS Param': 1})
```

Comment (ChrisTimperley) [reply geometry2/f2568f5/f2568f5.bug:49]: I'll open up a separate issue for this.

**Comment by ChrisTimperley** [comment]
If there's no objections, @gavanderhoorn, @ipa-hsd, let's go ahead and merge this into `master` and leave yamale/schema-related things out of this PR.

**Comment by gavanderhoorn** [comment]
Sure, I was just curious.

**Pull request merged**